android/src/main/java/com/reactnativecommunity/webview/RNCWebView.java

```diff
@@ -283,14 +314,16 @@ public void callInjectedJavaScriptBeforeContentLoaded() {
         }
     }
 
+    protected String injectedJavaScriptObject = null;
+
     public void setInjectedJavaScriptObject(String obj) {
         if (getSettings().getJavaScriptEnabled()) {
-            RNCWebViewBridge b = createRNCWebViewBridge(this);
-            b.setInjectedObjectJson(obj);
+            this.injectedJavaScriptObject = obj;
+            this.createRNCWebViewBridge(this);
```

Comment:
not a regression but an existing bug.
when testing on `examples/Injection` test case. i noticed that `setJavaScriptEnabled` coming after `setInjectedJavaScriptObject` (because we cannot ensure the order of prop updates). in this case, the injectedJavaScriptObject will not be updated because the `getJavaScriptEnabled()` is false here.

but previously it's just `window.ReactNativeWebView.injectedObjectJson()` returning null. now it would be `window.ReactNativeWebView.injectedObjectJson === undefined`

Reply:
makes sense, i've defaulted injectedObjectJson to returning null.